data/data-pipeline/data_pipeline/etl/score/constants.py

```diff
@@ -74,6 +74,17 @@
 ]
 TILES_ROUND_NUM_DECIMALS = 2
 
+TILES_ISLAND_AREAS = [
+    "American Samoa",
+    "Guam",
+    "Mariana Islands",
+    "US Virgin Islands",
+]
+TILES_ISLAND_FIELD_COUNT = 4
```

Comment:
Nit: "Field" is a bit vague, maybe call it "Thresholds" to match the language we use in the UI?

Reply:
also feel free to tell me to buzz off at any point with all my OCD nits ;)

Reply:
OCD nits are generally helpful imo (with the idea that hopefully each PR has fewer stylistic issues), so this sounds good